src/streaming/controllers/AlternativeMpdController.js

```diff
@@ -306,7 +306,7 @@ function AlternativeMpdController() {
                 maxDuration: alternativeMpdNode.maxDuration / timescale,
                 alternativeMPD: {
                     url: alternativeMpdNode.url,
-                    earliestResolutionTimeOffset: parseInt(alternativeMpdNode.earliestResolutionTimeOffset || '0', 10),
+                    earliestResolutionTimeOffset: parseInt(alternativeMpdNode.earliestResolutionTimeOffset || '60', 10),
```

Comment:
Add a const for this:

```suggestion
                    earliestResolutionTimeOffset: parseInt(alternativeMpdNode.earliestResolutionTimeOffset || 'DEFAULT_EARLIEST_RESOULTION_TIME_OFFSET', 10),
```